src/ReactWrapper.jsx

```diff
@@ -915,3 +916,14 @@ export default class ReactWrapper {
     unmountComponentAtNode(this.options.attachTo);
   }
 }
+
+
+if (ITERATOR_SYMBOL) {
+  Object.defineProperty(ReactWrapper.prototype, ITERATOR_SYMBOL, {
+    value: function iterator() {
```

Comment:
There's just no reason to lock it down - someone might want to override it for their project, or something.